config/src/test/java/org/axonframework/config/EventProcessingModuleTest.java

```diff
@@ -400,9 +401,9 @@ void configureSpanFactory() {
             GenericEventMessage<Object> message = new GenericEventMessage<>("test");
             config.eventBus().publish(message);
 
-            spanFactory.verifySpanCompleted("SubscribingEventProcessor[subscribing].process");
+            spanFactory.verifySpanCompleted("SubscribingEventProcessor.handle");
             assertWithin(2, TimeUnit.SECONDS,
-                         () -> spanFactory.verifySpanCompleted("TrackingEventProcessor[tracking].process"));
+                         () -> spanFactory.verifySpanCompleted("SubscribingEventProcessor.handle"));
```

Comment:
I did some digging. The assertion failed because there are two actions, both streaming and subscribing, at the same time and I got them confused.
The assertion failed as well, as it's no longer the same after retrieving it from storage. 

I have adjusted the assertion back to the correct and fixed the assertion method.